docs/00-introduction.ipynb

```diff
@@ -0,0 +1,523 @@
+{
```

Comment:
<p>I like this comparison between the traditional workflow and modern one! It'd make a good short post.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/eoapi-fedgeoday25-workshop/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>+1</p><p> </p><p>Although - there's also a hybrid perspective (as with CSDA) where we leverage certain layers of eoAPI, but data access still requires users to download data for local processing (or at least processing that is not in the browser.)</p>